proxy/src/proxy/copy_bidirectional.rs

```diff
@@ -98,7 +98,7 @@ pub(super) struct CopyBuffer {
     amt: u64,
     buf: Box<[u8]>,
 }
-const DEFAULT_BUF_SIZE: usize = 8 * 1024;
+const DEFAULT_BUF_SIZE: usize = 1024;
```

Comment:
May I get a bit more context, why do we want to do that?
8 * 1024 is a standard buffer size from tokio.

Reply:
Larger buffers offer more performance in theory. But for our usecase it might not be so significant. Given each passthrough task needs 2 of these buffers, it was using 16KB for each connection, even if they are idle. In some regions where we have upwards of 5k connections in passthrough per pod, that can add up to just below 100MB of memory